core/sdk-core/src/main/java/software/amazon/awssdk/core/http/HttpResponseHandler.java

```diff
@@ -40,6 +40,7 @@
                                                                                   X_AMZN_REQUEST_ID_HEADER_ALTERNATE)
                                                                               .collect(Collectors.toSet()));
     String X_AMZ_ID_2_HEADER = "x-amz-id-2";
+    String X_AMZN_QUERY_ERROR = "x-amzn-query-error";
```

Comment:
Do we need it here?

Reply:
Yes, I'm trying to follow the coding convention of the package here. i saw the other headers which are used by the same ErrorHandler class reside here. Thought it is a good idea. Let me know if you think it is better put it elsewhere.

Reply:
Can we move it to the class that uses it? It seems to be very specific to query protocol and this class is a generic handler class

Reply:
ok. will do